frontend/views/preferences/edit.html.erb

```diff
@@ -0,0 +1,57 @@
+<%
+  if params['global']
+    if params['repo']
+      title = I18n.t('preference.global')
+      message = I18n.t('preference.global_message')
+    else
+      title = I18n.t('preference.user_global')
+      message = I18n.t('preference.user_global_message')
+    end
+  else
+    if params['repo']
+      title = I18n.t('preference.repo')
+      message = I18n.t('preference.repo_message')
+    else
+      title = I18n.t('preference.user_repo')
+      message = I18n.t('preference.user_repo_message')
+    end
+    title += " -- #{current_repo['repo_code']}"
+  end
+%>
+ 
+<%= setup_context(:title => title) %>
+
+<%= form_for @preference, :as => "preference", :url => {:action => :update, :id => @preference.id}, :html => {:class => 'form-horizontal aspace-record-form'}.merge(update_monitor_params(@preference)) do |f| %>
+
+<% if params['global'] %>
+  <input type="hidden" name="global" value="true">
+<% end %>
+
+<% if params['repo'] %>
+  <input type="hidden" name="repo" value="true">
+<% end %>
+
+<%= form_context :preference, @preference do |form| %>
+
+<div class="row">
+   <div class="col-md-3">
+   </div>
+  <div class="col-md-9">
+    <%= render_aspace_partial :partial => "toolbar" %>
+    <div class="record-pane">
+      <h2><%= title %></h2>
+      <div class="alert alert-info">
+        <%= message.force_encoding("UTF-8") %>
+      </div>
+
+         <%= render_aspace_partial :partial => "shared/flash_messages" %>
+
+    <%= render_aspace_partial :partial => "shared/subrecord_form", :locals => {:form => form, :name => "defaults", :cardinality => :zero_to_one, :template => "defaults", :really_hidden => true} %>
+
+    </div>
+  </div>
+</div>
+
+<% end %>
+
+<% end %>
```

Comment:
yes, :hidden becomes :really_hidden

this was the only template, other than all the external_id ones that used :hidden

so changed this one and subrecord form instead of all the others.